config.lua

```diff
@@ -29,6 +29,10 @@ statusTimeout = 5000
 replaceKickOnLogin = true
 maxPacketsPerSecond = 25
 
+-- Store configuration
+storeImagesUrl = "http://static.tibia.com/images/store/"
```

Comment:
Ye change it to the ip located in config or something but keep the /images/.. if someone wants to use that URL.